examples/prometheusservice/rulegroupsnamespace.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: prometheusservice.aws.crossplane.io/v1alpha1
+kind: RuleGroupsNamespace
+metadata:
+  name: example-rule
+spec:
+  forProvider:
+    region: us-east-1
+    name: example-rule
+    workspaceID: ws-00000000-0000-0000-0000-000000000000
```

Comment:
can we use here workspaceIdRef?

Reply:
Yes, I am waiting your answer here https://github.com/crossplane-contrib/provider-aws/pull/1390#issuecomment-1197440184 And then I will use workspace id ref